spec/integration/users_spec.rb

```diff
@@ -0,0 +1,100 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :feature do
+  let(:person1) do
+    User.new(name: 'Bilal', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'I am full stack developer',
+             posts_counter: 0)
+  end
+
+  let(:person2) do
+    User.new(name: 'Ashna', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'I am full stack developer',
+             posts_counter: 0)
+  end
+
+  let(:first_post) do
+    Post.new(author_id: person1.id, title: 'Hello this is the first post', text: 'Most important description',
+             likes_counter: 0, comments_counter: 0)
+  end
+
+  let(:second_post) do
+    Post.new(author_id: person1.id, title: 'Hello-2 this is the second post', text: 'Most important description-2',
+             likes_counter: 0, comments_counter: 0)
+  end
+
+  before do
+    person1.save!
+    person2.save!
+    first_post.save!
+    second_post.save!
+  end
+
+  it 'Checks the name of users' do
+    visit root_path
+    expect(page).to have_content(person1.name)
+  end
+
+  it 'Checks for the number of posts each user has' do
+    visit root_path
+    expect(page).to have_content(person1.posts.count)
+  end
+
+  it 'Checks if the profile image successfully rendered for each user' do
+    visit root_path
+    expect(page).to have_css("img##{person1.id}")
+  end
```

Comment:
- [ ] We are testing for multiple users. This requires that we check for multiple `persons`. Kindly refactor these tests to demonstrate that we are testing for more than one user.

Reply:
Fixed.